k8s/utils/symphony-api.go

```diff
@@ -238,3 +239,20 @@ func CreateSymphonyDeployment(ctx context.Context, instance solution_v1.Instance
 
 	return ret, err
 }
+
+func NeedWatchInstance(instance solution_v1.Instance) bool {
```

Comment:
Why we need default time duration here? I think you can only check,
1. ReconciliationPolicy is inactive.
or 2. interval == 0.

Reply:
Just init with a non-zero value. I think the default is 0.